### PR TITLE
[AzCopyV10]  Add in content-md5 to available list properties

### LINKED ISSUE #2032: azcopy list --properties "ContentMD5" does not return md5sum of the file
### Which version of the AzCopy was used? 
azcopy version 10.16.0

### Which platform are you using? (ex: Windows, Mac, Linux)
MacOS Ventura 13.1

##### Note: Please remove the SAS to avoid exposing your credentials. If you cannot remember the exact command, please retrieve it from the beginning of the log file.
azcopy list https://<stgaccount/container> --properties "Content-MD5"

### What problem was encountered?
Would love to just return the md5sum property set for each file 

### How can we reproduce the problem in the simplest way?
Choose any file with Content-MD5 and try to use list

### Have you found a mitigation/solution?
No great solution to list all the files

### PR DESCRIPTION
**Pull request by vsmalladi**
Added "ContentMD5" as  new flag to '--properties' in the list command which takes ; separated list of properties and will display these fields in list command output.

Closes #2032

### CODE REVIEW

**Comment by nakulkar-msft** [comment]
Hi @vsmalladi, please submit review to dev branch

File: cmd/list.go [hunk 1]
```diff
@@ -177,6 +178,8 @@ func (cooked cookedListCmdArgs) processProperties(object StoredObject) string {
 			builder.WriteString(propertyStr + ": " + object.contentType + "; ")
 		case contentEncoding:
 			builder.WriteString(propertyStr + ": " + object.contentEncoding + "; ")
+		case contentMD5:
+			builder.WriteString(propertyStr + ": " + object.contentMD5 + "; ")
```
Comment (adreed-msft) [inline cmd/list.go:182]: Consider base64 encoding or using some standard way to display an MD5 to an end user

**Review by adreed-msft — CHANGES_REQUESTED** [review]

File: cmd/list.go (hunk 1 above)
Comment (vsmalladi) [reply cmd/list.go:182]: Good idea

**Comment by vsmalladi** [comment]
@adreed-msft any other changes needed?

**Review by nakulkar-msft — APPROVED** [review]

File: cmd/list.go [hunk 2]
```diff
@@ -54,6 +55,7 @@ const (
 	blobAccessTier   validProperty = "BlobAccessTier"
 	contentType      validProperty = "ContentType"
 	contentEncoding  validProperty = "ContentEncoding"
+	contentMD5		         validProperty = "md5"
```
Comment (adreed-msft) [inline cmd/list.go:58]: This name does not align with the helptext. Otherwise, looks good.

**Review by adreed-msft — APPROVED** [review]

File: cmd/list.go (hunk 2 above)
Comment (adreed-msft) [reply cmd/list.go:58]: Also, formatting here.

Comment (vsmalladi) [reply cmd/list.go]: Fixed

**Comment by vsmalladi** [comment]
Thanks @adreed-msft and @nakulkar-msft made necessary changes, should be good to go now if you approve.

**Review by vibhansa-msft — APPROVED** [review]

**Comment by gapra-msft** [comment]
Hi @vsmalladi, Could you please merge latest dev into your PR? That should resolve some of the failing checks.

**Comment by vsmalladi** [comment]
@gapra-msft Should be good now.

**Pull request merged**